content/rest-api/builds.md

```diff
@@ -56,8 +62,43 @@ APIs for managing builds are currently available for developers to preview. Duri
 }
 ```
 
+## Get build status

```

Comment:
currently, this endpoint is not public and we don't guarantee backward compatibility. Can you share more details about your use case to consider making the endpoint public?

Reply:
Currently, there is an endpoint for running new builds, but there is no way for tracking the build progress using the API. So, this would be helpful for tracking the builds and getting updates as it completes. And the user can perform some other operations as the build finishes.